classroom.rb

```diff
@@ -0,0 +1,18 @@
+class Classroom
+  attr_accessor :label
+
```

Comment:
- [ ] You need to make the student readable/getter by setting the getter `attr_reader :student`

Reply:
* This issue has not been fixed yet, kindly look into it